Packages/com.unity.render-pipelines.universal/Shaders/PostProcessing/UberPost.shader

```diff
@@ -40,6 +41,13 @@ Shader "Hidden/Universal Render Pipeline/UberPost"
             #endif
         #endif
 
+
+        #if SUBPASS_INPUT_ATTACHMENT
+            #define urp_cameraColor 0
+            FRAMEBUFFER_INPUT_HALF_MS(urp_cameraColor);
```

Comment:
When this is enabled, do we also have the _BlitTexture bound to the shader as well?

Reply:
From the renderdoc I did not see that _BlitTexture anymore